sdk/java-sdk/src/main/java/kalix/javasdk/action/Action.java

```diff
@@ -148,6 +148,22 @@ interface Builder {
        * @param <S> The type of the message that must be returned by this call.
        */
       <S> Effect<S> asyncEffect(CompletionStage<Effect<S>> futureEffect);
+
+      /**
+       * Creates an empty response for an endpoint
+       * @return
+       * @param <S>
+       */
+      <S> Effect<S> empty();
+
+      /**
+       * Creates a response that ignores the current element of a subscription or a publication
+       * so that consumption or production can proceed to the next element
+       *
+       * @return
+       * @param <S>
+       */
+      <S> Effect<S> ignore();
```

Comment:
@johanandren what do you think?

Reply:
Looks good, did you double check that it causes 404/NotFound for a regular request-response? If so I think we can document that as well.

Reply:
I did not test it. Although I would expect a 200 OK with an empty response. I'll test it now

Reply:
Crystal clear now 👍 . I'll have a look at the proxy as well.

Reply:
https://github.com/lightbend/kalix-proxy/pull/1664